wasi/src/test/java/com/dylibso/chicory/wasi/WasiPreview1Test.java

```diff
@@ -77,4 +88,46 @@ public void shouldRunTinyGoModule() {
 
         assertEquals(result.asInt(), 42);
     }
+
+    @Test
+    public void shouldRunPythonModule() throws Exception {
+        // implementation of this tutorial:
+        var fakeStdout = new MockPrintStream();
+        var fakeStderr = new MockPrintStream();
+        FileSystem fs =
+                Jimfs.newFileSystem(
```

Comment:
Should we add an `<optional>true</optional>` dependency on Jimfs and add a utility method for this?